packages/system/kubeovn/charts/kube-ovn/templates/central-deploy.yaml

```
@@ -40,15 +40,42 @@ spec:
+            - mountPath: /var/run/ovn
+              name: host-run-ovn
+            - mountPath: /etc/ovn
+              name: host-config-ovn
+            - mountPath: /var/log/ovn
+              name: host-log-ovn
       containers:
         - name: ovn-central
           image: {{ .Values.global.registry.address }}/{{ .Values.global.images.kubeovn.repository }}:{{ .Values.global.images.kubeovn.tag }}
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           command: 
+          - bash
```

yamllint (1.35.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)